cmd/operator-sdk/add/api.go

```diff
@@ -69,6 +80,53 @@ Example:
 	$ tree deploy/crds
 	├── deploy/crds/app.example.com_v1alpha1_appservice_cr.yaml
 	├── deploy/crds/app.example.com_appservices_crd.yaml
+
+Ansible Example:
+
+	$ operator-sdk add api --api-version=cache.example.com/v1alpha1 --kind=Mykind	
+	$ tree ./roles
+	./roles
+		├── memcached
+		│   ├── README.md
+		│   ├── defaults
+		│   │   └── main.yml
+		│   ├── tasks
+		│   │   └── main.yml
+		├── mykind
+		│   ├── README.md
+		│   ├── defaults
+		│   │   └── main.yml
+		│   ├── tasks
+		│   │   └── main.yml
+Helm Example:
+	
+	$ operator-sdk add api --api-version=cache.example.com/v1alpha1 --kind=Mongodb --helm-chart=stable/mongodb
+	$ tree ./helm-charts/
+	./helm-charts/
+		├── memcached
+		│   ├── Chart.yaml
+		│   ├── README.md
+		│   ├── templates
+		│   │   ├── NOTES.txt
+		│   │   ├── _helpers.tpl
+		│   │   ├── pdb.yaml
+		│   │   ├── statefulset.yaml
+		│   │   └── svc.yaml
+		│   └── values.yaml
+		── mongodb
+		│   ├── Chart.yaml
+		│   ├── charts
+		│   ├── templates
+		│   │   ├── NOTES.txt
+		│   │   ├── _helpers.tpl
+		│   │   ├── deployment.yaml
+		│   │   ├── ingress.yaml
+		│   │   ├── service.yaml
+		│   │   ├── serviceaccount.yaml
+		│   │   └── tests
+		│   │       └── test-connection.yaml
+		│   └── values.yaml
+
```

Comment:
IMO we should not add te tree in the examples if something changed the command will start to scaffold the new layout and then it means that it get outdated easily.

Reply:
Agree, May be these details can go into examples later.